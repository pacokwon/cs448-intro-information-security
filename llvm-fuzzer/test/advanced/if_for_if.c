<<<<<<< HEAD
=======
#include <stdio.h>
#include <stdlib.h>

>>>>>>> fuzzer/master
int main()
{
    int x = 3;
    int i = 0;
    int j = 0;
<<<<<<< HEAD
=======
    char input[65536];
    fgets(input, sizeof(input), stdin);
>>>>>>> fuzzer/master

    if(x >= 0)
    {
        for(i = 0; i < x; i++)
        {
<<<<<<< HEAD

            for(j = 0; j < i; j++)
            {
                int m = (i * j) / i;
=======
            for(j = 0; j < i; j++)
            {
                int m = (i * j) / atoi(input);
>>>>>>> fuzzer/master

                if(m >= 0)
                {
                    int k = x / m;
                }
            }
        }
    }
<<<<<<< HEAD
    else
    {
        x = -x;

        for(i = 0; i < x; i++)
        {

            for(j = 0; j > i; j--)
            {
                int m = (i * j) / i;

                if(m >= 0)
                {
                    int k = x / m;
                }
            }
        }
    }

=======
>>>>>>> fuzzer/master
}
