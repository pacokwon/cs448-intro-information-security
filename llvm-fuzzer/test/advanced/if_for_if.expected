Divide-by-zero detected at line 18 and col 31
